### ISSUE TITLE
Precook tweets

### ISSUE THREAD

**Issue by sverhoeven** [NLESC-JCER/run-cpp-on-web#73]
To spread word of the published blogs we need to make some posts to social media.  Let's prepare some tweets.

**Comment by sverhoeven** [comment]
Some suggestions:

1
Want to learn how to run your C++ code on the web. See https://medium.com.... for a intro into web technologies to make this happen

2
Ever had some C++ code you wanted to show and run for a wider community. Then see https://medium.com/... for the steps to create an interactive web application with visualizations running C++ code in a web browser

3
Researchers who want to make a web application from their C++ code rejoice.  https://medium.com/... series of blogs has been published to explain the steps to do this. Great for demonstration or to move the heavy computation from the server to the browser.

4
All these web technologies I do not know where to start. https://medium.com/... explains how to take your C++ code and make it into an web application using WebAssembly, Web workers, React and Vega.

**Comment by cwmeijer** [comment]
I like your third option although the first sentence is a bit hard to parse and a bit long. 

Combining the first sentence of 1 with the rest of 3:

_Want to learn how to run your C++ code on the web?  https://medium.com/... series of blogs has been published to explain the steps to do this. Great for demonstration or to move the heavy computation from the server to the browser._

234 characters, 41 words. Is that too long for a tweet?

**Comment by sverhoeven** [comment]
Tweet has been scheduled
